lib/siwapp/settings.ex

```diff
@@ -23,6 +23,18 @@ defmodule Siwapp.Settings do
   """
   def prepare_data, do: struct(SettingBundle, Enum.zip(SettingBundle.labels(), values()))
 
+  def prepare_data(:cache) do
+    case Cachex.get(:my_cache, :settings_data) do
+      {:ok, nil} ->
+        settings_data = prepare_data()
```

Comment:
Como no cambiéis pronto ese prepare_data voy a terminar bloqueando cualquier PR que lo use

Reply:
ok